test/storage.test.js

```diff
@@ -4,6 +4,11 @@ describe('Tests for storage', () => {
   const feed = { name: 'James Smith', url: 'http://seneca.co/jsmith' };
   const feed2 = { name: 'James Smith 2', url: 'http://seneca.co/jsmith/2' };
 
+  it('check that storage is empty', async () => {
+    const result = await storage.getFeedsCount();
```

Comment:
This test is risky, since it depends on the order tests get run in, and could fail if that ever changed.

A better way would be to get the count, add a feed, get the count again, check that it's greater by 1.